grpc-client.go

```diff
@@ -182,10 +183,18 @@ func newFaithfulClient(serverAddr string, useTLS bool) (*faithfulClient, error)
 
 	// Set maximum message size (100MB)
 	opts = append(opts, grpc.WithDefaultCallOptions(
-		grpc.MaxCallRecvMsgSize(100*1024*1024),
-		grpc.MaxCallSendMsgSize(100*1024*1024),
+		grpc.MaxCallRecvMsgSize(100*MiB),
+		grpc.MaxCallSendMsgSize(100*MiB),
 	))
 
+	// Add client-side keepalive parameters
+	keepaliveParams := keepalive.ClientParameters{
+		Time:                30 * time.Second, // Ping server if idle for 30 seconds
```

Comment:
Let's change this to 60s.
```suggestion
		Time:                60 * time.Second, // Ping server if idle for 30 seconds
```